vars/pipeline2ATX.txt

```diff
@@ -5,6 +5,9 @@ The report is compressed as a zip file within the build workspace and can be upl
 <ul>
 <li><a href="https://plugins.jenkins.io/pipeline-utility-steps/">Pipeline Utility Steps Plugin</a></li>
 <li><a href="https://plugins.jenkins.io/pipeline-stage-view/">Pipeline Stage View Plugin</a></li>
+<li>Environment variable "TEST_LEVEL" with a string of the corresponding test level</li>
```

Comment:
There is a mismatch between the variables documentation TEST_LEVEL and its implementation